pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -258,6 +258,8 @@
     // Name of load manager to use
     @FieldContext(dynamic = true)
     private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
+    @FieldContext(dynamic = true)
+    private boolean preferLaterVersions = true;
```

Comment:
Also, we should add a brief comment line to explain what the switch does and add the same in `conf/broker.conf`.